src/components/content/ArticlePage.tsx

```diff
@@ -55,7 +60,7 @@ export default function ArticlePage({ data, contentId, contentType }) {
           <FontAwesomeIcon icon={faShareAlt} size="1x" /> Teilen
         </ToolLineButton>
       </ToolLine>
-      {data.value && renderArticle(data.value.children)}
+      {data.value && renderArticle(data.value.children, false, prettyLinks)}
```

Comment:
We should refactor that into an object parameter for readability.

Reply:
something like?
``` tsx
{data.value && renderArticle({children: data.value.children, parameter: false, prettyLinks: prettyLinks)}
```

Reply:
Yep, except maybe a more descriptive name for parameter (not sure what it does currently). IIRC, the second parameter was optional anyways (so you can remove it from your call again)